htdocs/landing/index.php

```diff
@@ -21,7 +21,27 @@ <h1 class="Landing_Welcome">Welcome to GOCDB</h1>
             <p>Use of GOCDB is governed by the <a class="docLink hover" href="aup.html">EGI Acceptable Use Policy</a> which places restrictions on your use of the service.</p>
             <p>The <a class="docLink hover" href="privacy.html">GOCDB Privacy Notice</a> describes what personal data is collected and why, and your rights regarding this data.</p>
             <p> Please read these documents before accessing GOCDB.</p>
-            <a href="/portal/" class="button">Access GOCDB</a>
+          </div>
+          <div style="width: 80%; margin-left: auto; margin-right: auto;">
+            <a href="/portal/" style="width:73%; font-size:1.7em" class="button">Access GOCDB via X.509 Certificate</a>
```

Comment:
```suggestion
            <a href="/portal/" style="width:73%; font-size:1.7em" class="button">Access GOCDB using your IGTF X.509 Certificate</a>
```

Reply:
Changes the size of the button as well to better format with new text

```suggestion
            <a href="/portal/" style="width:68%; font-size:1.7em" class="button">Access GOCDB using your IGTF X.509 Certificate</a>
```